test/test-cases/functional/saic/sai-api/test_sai_api_vnet_direction_lookup.py

```diff
@@ -0,0 +1,108 @@
+import json
+import time
+from pathlib import Path
+from pprint import pprint
+
+import pytest
+
+# Constants
+SWITCH_ID = 5
+
+
+class TestSaiDirectionLookup:
+
+    def test_direction_lookup_create(self, dpu):
+
+        commands = [
+            {
+                "name": "direction_lookup_entry",
+                "op": "create",
+                "type": "SAI_OBJECT_TYPE_DIRECTION_LOOKUP_ENTRY",
+                "key": {
+                    "switch_id": "$SWITCH_ID",
+                    "vni": "2000"
+                },
+                "attributes": [
+                    "SAI_DIRECTION_LOOKUP_ENTRY_ATTR_ACTION",
+                    "SAI_DIRECTION_LOOKUP_ENTRY_ACTION_SET_OUTBOUND_DIRECTION"
+                ]
+            },
+        ]
+        results = [*dpu.process_commands(commands)]
+        print("\n======= SAI commands RETURN values create =======")
+        pprint(results)
+
+        assert all(results), "SAI_OBJECT_TYPE_DIRECTION_LOOKUP_ENTRY Create error"
+
+    @pytest.mark.skip(reason="get and set not implemented, yet")
+    def test_direction_lookup_get1(self, dpu):
+
+        commands = [
+            {
+                "name": "direction_lookup_entry",
+                "op": "get",
+                "type": "SAI_OBJECT_TYPE_DIRECTION_LOOKUP_ENTRY"
+            }
+        ]
+        results = [*dpu.process_commands(commands)]
+        print("\n======= SAI commands RETURN values get =======")
+        pprint(results)
+
+        assert all( [result == 0 for result in results]), "2000"
```

Comment:
What does "2000" means. This should be a message which is displayed in case of failure and help to understand the failure reason.